functions/Get-DbaInstallDate.ps1

```diff
@@ -0,0 +1,188 @@
+
+Returns an object with SQL Server Install date as a string and the Windows install date as a string for both SQLInstances that are passed to the cmdlet.  
+	
+.EXAMPLE   
+Get-DbaInstallDate -SqlServer sqlserver2014a, sql2016 -SqlOnly
+
+Returns an object with only the SQL Server Install date as a string. 
+
+.EXAMPLE   
+Get-DbaInstallDate -SqlServer sqlserver2014a, sql2016 -WindowsOnly
+
+Returns an object with only the Windows Install date as a string. 
+
+.EXAMPLE   
+Get-SqlRegisteredServerName -SqlServer sql2014 | Get-DbaInstallDate
+
+Returns an object with SQL Server Install date as a string and the Windows install date as string for every server listed in the Central Management Server on sql2014
+	
+#>
+	[CmdletBinding(DefaultParameterSetName = "Default")]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance", "ComputerName")]
+		[object[]]$SqlServer,
+		[parameter(ParameterSetName = "Sql")]
+		[Switch]$SqlOnly,
+		[parameter(ParameterSetName = "Windows")]
+		[Switch]$WindowsOnly,
+		[Alias("Credential")]
+		[PsCredential]$SqlCredential,
+		[PsCredential]$WindowsCredential
+	)
+	
+	PROCESS
+	{
+		foreach ($instance in $SqlServer)
+		{
+			if ($instance.Gettype().FullName -eq [System.Management.Automation.PSCustomObject] )
+			{
+				$servername = $instance.SqlInstance
+			}
+			elseif ($instance.Gettype().FullName -eq [Microsoft.SqlServer.Management.Smo.Server])
+			{
+				$servername = $instance.NetName
+			}
+			else
+			{
+				$servername = $instance
+			}
+						
+			if ($WindowsOnly -ne $true)
+			{ 
+				try 
+				{
+					Write-Message -Level Verbose -Message "Connecting to $instance" -Silent $false 
+                    if ($SqlCredential)
+                    { 
+                        $server = Connect-SqlInstance -SqlInstance $servername -SqlCredential $SqlCredential #-ErrorVariable ConnectError
+                    } else 
+                    { 
+                        $server = Connect-SqlInstance -SqlInstance $servername 
+                    }					
+				}
+				catch 
+				{
+					Stop-Function -Message "Failed to connect to: $instance" -Continue -Target $instance					
+				}
+
+				if ( $server.VersionMajor -ge 9 )
+				{ 
+					Write-Message -Level Verbose -Message "Getting Install Date for: $instance" -Silent $false 
+					$sql = "SELECT create_date FROM sys.server_principals WHERE sid = 0x010100000000000512000000"
+					$sqlInstallDate = $server.databases['master'].ExecuteWithResults($sql).tables
+					$sqlInstallDate = ($sqlInstallDate.rows).create_date.toString('MM-dd-yyyy') 
+
+				} else { 
+					Write-Message -Level Verbose -Message "Getting Install Date for: $instance" -Silent $false 
+					$sql = "SELECT schemadate FROM sysservers"
+					$sqlInstallDate = $server.databases['master'].ExecuteWithResults($sql).tables					
+					$sqlInstallDate = ($sqlInstallDate.rows).create_date.toString('MM-dd-yyyy') 
```

Comment:
This should be returned as a date and not just a string. Not all locales will show dates as `MM-dd-yyyy`. 

We have a date type built in as `SqlCollective.dbatools.Utility.DbaDateTime` that can be used for this. Should be able to just do it as:
```
[SqlCollective.dbatools.Utility.DbaDateTime]$sqlInstallDate = $sqlInstallDate.Rows.Create_Date
```